### DISCUSSION TITLE
finished project

### DISCUSSION THREAD

**Discussion by ademsenel** [Dotnet-Boxed/Templates#81] [Q&A]
Hi Rehan

My finished Project address is http://www.adddemo.com/postakodu/ 
But i haven't interested some features supported in your Boilerplate mvc5 template 
i change some feature so i used ninject instead of autofac, and i build project cruid 
thank you for your gereat projects

**Comment by ademsenel** [comment]
Can you check and report if any issue pls

**Comment by RehanSaeed** [comment]
Cool, would you like me to add it to the home page?

**Comment by ademsenel** [comment]
yes, why not i would be glad if you add

**Comment by RehanSaeed** [comment]
Done. Glad you found the template useful. I can't understand the language but it's nice to see that you've linked to Boilerplate too.
